### ISSUE TITLE
Unable to edit methods due to never ending decompilation

### ISSUE THREAD

**Issue by STALKER-medved** [0xd4d/dnSpy#1033]
Before clicking `Submit new issue`, try the [latest build](https://ci.appveyor.com/project/0xd4d/dnspy/branch/master/artifacts)

- What version did you use?

- Write down all steps needed to reproduce this issue. Also attach any needed files.

**Comment by STALKER-medved** [comment]
When trying to edit a method in Assembly-CSharp.dll in a game file, I am greeted by the decompiling... loading screen and is stuck forever in decompilation unless I click cancel.

**Comment by STALKER-medved** [comment]
Happy new year in the Slavic way.

**Comment by 0xd4d** [comment]
It's probably a big method. Can you view this method at all?

You can switch to IL view and use the IL editor.

**Comment by STALKER-medved** [comment]
I am able to view the method and it is about 23 lines of code, and I tried IL editor and it also had the same issue.

**Comment by 0xd4d** [comment]
Can you attach the files here?

**Comment by STALKER-medved** [comment]
I think this has to do with me putting disputed on a usb and then putting it into my computer other than that I don’t really know.

**Comment by Tianjiao** [comment]
@0xd4d Files may contain warez or malware(not this case necessarily), you get little compared with the risk.
 If you wanna repro this, log the debug info then tell user to send the log. 
Don't become [honeypot](https://en.wikipedia.org/wiki/Honeypot_(computing)) yourself.

**Comment by Tianjiao** [comment]
@0xd4d PS: If you're tired of old Issues, why not give GitHub [Robot](https://probot.github.io/) a try? It can close issues older than 2 months, or any period you set, based on your label. So that it saves your time to focus on developing, and keep equal standard among your issues.

**Comment by Tianjiao** [comment]
@STALKER-medved Sorry I didn't mean you but @0xd4d often request files from unknown source. Think of Gmail, the chance of e-mail communication is low, but once it hit upon you, then the chance is 100%, and it needs only once. That's the reason I never ask my user for file. I log the debug info instead.

**Comment by STALKER-medved** [comment]
Here are the photos you requested.
![b3fc52ba-9630-4f5a-8830-80ed3c9f83f2](https://user-images.githubusercontent.com/46289989/50610159-c977f800-0e86-11e9-99fa-884f9c96727d.jpeg)
![a930b12d-3b1e-4248-8f67-d04008c46c0d](https://user-images.githubusercontent.com/46289989/50610160-c977f800-0e86-11e9-8a84-d36afc822e6e.jpeg)
![32ad20ae-68e5-477d-94af-34367a04b45d](https://user-images.githubusercontent.com/46289989/50610161-c977f800-0e86-11e9-8a63-80ced67f93e2.jpeg)

**Comment by STALKER-medved** [comment]
And it decompiles forever until I click cancel

**Comment by STALKER-medved** [comment]
I know the photos are low quality because my computer can’t acces the internet, so I had to download dnspy from a friends laptop, put dnspy on a USB, and then put it into my computer.

**Comment by STALKER-medved** [comment]
Not mention that why I’m using an iPad for this.

**Comment by STALKER-medved** [comment]
The method I’m editing is WeaponManager, and in the weapon manager I’m editing UnlockSecretWeapon.

**Comment by 0xd4d** [comment]
This isn't enough to figure out what's happening. Have you tried it on some other computer? Otherwise attach the files here.

**Comment by STALKER-medved** [comment]
Like I said before, I don’t know the cause of the problem, it should work normally right? Is this a common thing that happens when you first open dnspy?

**Comment by STALKER-medved** [comment]
And yes I tried it on another system, same thing, I think it has to do with the .zip file being moved to a usb or something.

**Comment by STALKER-medved** [comment]
i tried pretty much every decompiler there ilspy, dotspeek, visual studio,etc, please help!

**Comment by 0xd4d** [comment]
Provide some more info.

> i tried pretty much every decompiler there ilspy, dotspeek, visual studio,etc, please help!

So you tried them and what happened? That info is completely missing.
